Dockerfile

```
@@ -1,4 +1,4 @@
-FROM fedora:41
+FROM fedora:42
 LABEL org.opencontainers.image.source="https://github.com/GSConnect/gsconnect-ci" org.gsconnect.gnome.version="47"
```

(This is a sign I should probably remove this tag attempt, as it doesn't seem to WORK anyway,and means that it has to be kept updated independent of dependabot.)

```suggestion
LABEL org.opencontainers.image.source="https://github.com/GSConnect/gsconnect-ci" org.gsconnect.gnome.version="48"
```